test/AppStatsTest.php

```diff
@@ -31,6 +31,8 @@ protected function setUp() {
     }
 
     public function testPublishEventsForwards() {
+        echo '== testPublishEventsForwards()';
```

Comment:
I believe all these echoes are there just to see how phpunit is progressing, otherwise it outputs just "....."